### DISCUSSION TITLE
The "save" button on draft items is not large/obvious enough

### DISCUSSION THREAD

**Discussion by torrobinson** [community/community#9946] [Projects and Issues]
When typing a new draft item into a board, and then clicking on the title:
![image](https://user-images.githubusercontent.com/4401999/148816543-bff3662d-b45a-4786-972d-cd3bd5657a92.png)

A side-panel opens up for editing it, with the description ready to type in:
![image](https://user-images.githubusercontent.com/4401999/148816655-dfa7a22d-6842-4703-ad31-c6b4eca458ea.png)

After adding some text, I went to convert the draft to an issue:
![image](https://user-images.githubusercontent.com/4401999/148816756-3815b0e5-5d98-44db-80c0-e4eec21822b6.png)

and was warned about unsaved changes:
![image](https://user-images.githubusercontent.com/4401999/148816788-2bbb21b1-1f88-4d4f-ac48-5972285b4b42.png)

In my opinion, this save button is too small:
![image](https://user-images.githubusercontent.com/4401999/148816810-283a98a5-dc79-4f80-a352-e202c560bfb7.png)

, as I had to scan the screen for a few seconds to see that it was considered unsaved.

I do appreciate the need to save and the warning (I dislike when an app auto-saves but doesn't tell you and the save state is unknown), but I think perhaps making the Save button more obvious might help? 

Perhaps giving it a background: 
![image](https://user-images.githubusercontent.com/4401999/148817013-8a9a7373-b4e3-4fc8-926b-2c8f428e7b51.png)

or adding it to the bottom for the form, where save/cancel buttons are more common, might help?

**Comment by jaybuidl** [comment]
Same, just lost 30 minutes of work

**Comment by Farr69** [comment]
> When typing a new draft item into a board, and then clicking on the title: ![image](https://user-images.githubusercontent.com/4401999/148816543-bff3662d-b45a-4786-972d-cd3bd5657a92.png)
> 
> A side-panel opens up for editing it, with the description ready to type in: ![image](https://user-images.githubusercontent.com/4401999/148816655-dfa7a22d-6842-4703-ad31-c6b4eca458ea.png)
> 
> After adding some text, I went to convert the draft to an issue: ![image](https://user-images.githubusercontent.com/4401999/148816756-3815b0e5-5d98-44db-80c0-e4eec21822b6.png)
> 
> and was warned about unsaved changes: ![image](https://user-images.githubusercontent.com/4401999/148816788-2bbb21b1-1f88-4d4f-ac48-5972285b4b42.png)
> 
> In my opinion, this save button is too small: ![image](https://user-images.githubusercontent.com/4401999/148816810-283a98a5-dc79-4f80-a352-e202c560bfb7.png)
> 
> , as I had to scan the screen for a few seconds to see that it was considered unsaved.
> 
> I do appreciate the need to save and the warning (I dislike when an app auto-saves but doesn't tell you and the save state is unknown), but I think perhaps making the Save button more obvious might help?
> 
> Perhaps giving it a background: ![image](https://user-images.githubusercontent.com/4401999/148817013-8a9a7373-b4e3-4fc8-926b-2c8f428e7b51.png)
> 
> or adding it to the bottom for the form, where save/cancel buttons are more common, might help?

Did you found any solution? I am facing the same issue.
[Indigo Card](https://www.indigocard.biz/)

**Comment by pcholakov** [comment]
In addition, the `cmd+return` shortcut doesn't work.
